lua/modules/controls.lua

```diff
@@ -482,7 +482,7 @@ local _setPointerUp = function(pointerIndex, pointerEvent)
 		end
 	end
 
-	if _state.touchDragPointer ~= nil and _state.touchDragPointer.index == pointerIndex then
```

Comment:
With the second press, the `pointerIndex` is not the one we want if the last finger doesn't move, it is strange because it is not altered in our code base and happens on Android and iOS